### DISCUSSION TITLE
Automatically encapsulate data list with single-quotes and commas

### DISCUSSION THREAD

**Discussion by Austin1** [dbeaver/dbeaver#15938] [Ideas]
I don't believe this is already a feature, but my google-fu could be failing me.

A feature I've seen in other products that is a great feature for analysts is being able to quickly take a list of data and paste it into the SQL Editor and highlight it and press a hotkey to encapsulate the list with single quotes and commas.

The common workflow is you have 100 records in an excel sheet that you want to include in a "in" statement of SQL. You paste it into the editor, highlight it, and then press something that adds the single quotes and commas for it to work. It's not incredible difficult to do in excel with concatenation but it's a really nice to have. Just an idea!

**Comment by p91paul** [comment]
Try right click -> format -> morph into delimited list. You can probably bind this to a shortcut in preferences!

**Reply by Austin1** [reply]
well this is magical. Maybe this can server as a google answer for someone. Exactly what I was trying to find!

**Reply by Arnon-T** [reply]
It did.
